doc/contributing/test.rst

```diff
@@ -105,7 +105,7 @@ To enable multi-core:
          <core name="ckan" instanceDir="/etc/solr/ckan" />
        </cores>
 
-5. Restart Solr by restarting Jetty (or Tomcat)::
+5. Restart Solr by restarting Jetty (may also be called jetty8) (or Tomcat)::
```

Comment:
Here service referred only in general, not by its internal name. If you are going to list all possible names of jetty/tomcat service it will take a lot of space. In addition, if you are going to use some other Linux distributive, not Ubuntu, the number of choices will increase, so it has almost no sense to mention `jetty8` separately. But you may add note about the fact, that service names may be different on different systems

Reply:
i think `your jetty executable may have a different name, eg: jetty8 or jetty9` - is perfect, because it doesn't make restriction to single possible alternative